src/pages/ReservationPage.jsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchReservations, removeReservation } from '../redux/reservations/reservationSlice';
+import './ReservationPage.css';
```

Comment:
this css route has a typo. it should be `./reservationPage.css` (first R is small case)